hr_announcement/models/announcement.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class Announcement(models.Model):
+    _inherit = "announcement"
+
+    announcement_type = fields.Selection(
+        selection_add=[
+            ("employee", "Employee"),
+            ("department", "Department"),
+            ("job_position", "Job Position"),
+        ],
+        ondelete={
+            "employee": "set default",
+            "department": "set default",
+            "job_position": "set default",
+        },
+    )
+
+    employee_ids = fields.Many2many(
+        comodel_name="hr.employee",
+        string="Employees",
+        inverse="_inverse_employee_ids",
```

Comment:
From my lack of knowledge in version 15.0, is it possible to apply an `inverse=`
 to a field that is not `compute=`? 

Wouldn't it be cleaner to launch a trigger in the write/create.

Reply:
You are right, thank you for your corrections